scripts/zulip-puppet-apply

```diff
@@ -7,15 +7,10 @@ import subprocess
 import sys
 import tempfile
 
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-sys.path.append(BASE_DIR)
-from scripts.lib.setup_path import setup_path
-from scripts.lib.zulip_tools import assert_running_as_root, parse_os_release
-
-setup_path()
-
 import yaml
```

Comment:
There's an argument that we should only import this in the code path that actually uses it; could save ~50ms on startup time for this script.  `puppet` is slow enough that I'm not sure it's worth that sort of optimization for this script in particular.